npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -69,6 +69,22 @@ function modifyWebpackConfigForCypress (webpackConfig: Partial<Configuration>) {
     }
   }
 
+  if (webpackConfig?.module?.rules) {
+    let newRules: (RuleSetRule | '...')[] = []
+
+    webpackConfig.module.rules.forEach((ele: any) => {
+      const regex = new RegExp(ele.test)
+
+      if (regex.test('test.css')) {
```

Comment:
Does this work if they are using `.scss` or another flavor os css like stylus or less?

Reply:
You are right. This does not work for other css file extensions like .less/.scss. So the more I think about this I now realize that there could be an inordinate amount of regex expressions that a user can specify in his config, potentially, one that we haven't accounted for in our conditions.